tensorflow/lite/experimental/micro/kernels/all_ops_resolver.cc

```diff
@@ -29,12 +29,18 @@ TfLiteRegistration* Micro_Register_FULLY_CONNECTED() {
 TfLiteRegistration* Register_SOFTMAX();
 TfLiteRegistration* Micro_Register_SOFTMAX() { return Register_SOFTMAX(); }
 
+TfLiteRegistration* Register_MAX_POOL_REF();
+TfLiteRegistration* Micro_Register_MAX_POOL_REF() {
```

Comment:
Since other opts don't have _REF suffix for REF impl. Probably worth keeping it consistent.